### ISSUE TITLE
Linux Headless Crashes on World Record Sync

### ISSUE THREAD

**Issue by PointerOffset** [Yellow-Dog-Man/Resonite-Issues#1135]
### Describe the bug?

After saving changes to a world on the Headless client, the client will crash as it attempts to sync the updated record. 

Version: 2024.1.12.1336, 2024.1.13.8
Platform: Linux (Docker Container)
Reproduction Rate: 100%

I tried saving an established world being worked on and a fresh grid space world I created to test reproduction.

### To Reproduce

1. Start up a Headless session with a record the headless user has permission to save.
2. Run the "save" command on the Headless terminal.
3. After save completes successfully, wait a few moments for the record sync to start.
4. Crash

### Expected behavior

When saving a world on a Headless client the world record should sync successfully and the client should continue running.

### Resonite Version Number

2024.1.12.1336, 2024.1.13.8

### What Platforms does this occur on?

Linux

### Log Files

My log files weren't being saved where I expected so I don't have a full log to provide yet. I will add one as soon as I'm able but this is the exception the terminal provides to me:

```
[ERROR] FATAL UNHANDLED EXCEPTION: System.IO.InvalidDataException: Compression failed
  at BrotliSharpLib.BrotliStream.WriteCore (System.Byte[] buffer, System.Int32 offset, System.Int32 count, BrotliSharpLib.Brotli+BrotliEncoderOperation operation) [0x00099] in <01c8bc3595934f69929dc84cde233830>:0 
  at BrotliSharpLib.BrotliStream.FlushCompress (System.Boolean finish) [0x00027] in <01c8bc3595934f69929dc84cde233830>:0 
  at BrotliSharpLib.BrotliStream.Dispose (System.Boolean disposing) [0x00008] in <01c8bc3595934f69929dc84cde233830>:0 
  at BrotliSharpLib.BrotliStream.Finalize () [0x00000] in <01c8bc3595934f69929dc84cde233830>:0
```

### Additional Context

- This reproduced 100% of the time under my Linux container host.
- This did not reproduce on Windows. 
- I could not get the Linux Headless running under bare-metal to test the issue.

### Reporters

GitHub: @PointerOffset, Discord: @spexcat

**Comment by Frooxius** [comment]
Same issue as here: https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/1134

Brotli has problem under Linux for some reason. Currently investigating.

**Comment by Frooxius** [comment]
This should be fixed by 2024.1.13.83 too. Can you check please?

**Comment by PointerOffset** [comment]
Yup! This is now resolved. Thank you for the quick response!
